examples/README.md

```diff
@@ -1433,6 +1433,26 @@ spec:
 
 Resources created in this way are independent of the workflow. If you want the resource to be deleted when the workflow is deleted then you can use [Kubernetes garbage collection](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/) with the workflow resource as an owner reference ([example](./k8s-owner-reference.yaml)).
 
+You can also collect data about the resource in output parameters (see more at [k8s-jobs.yaml](./k8s-jobs.yaml#L42-L56)):
+
```

Comment:
This is good to have but I wonder whether this is in scope as there are many other advanced examples in https://github.com/argoproj/argo-workflows/tree/master/examples but not mentioned either in this README.md. @alexec WDYT?

Reply:
The problem is that the example is about k8s-jobs whereas the concept applies to all k8s resources

And so, it is not obvious to open this exact example in this context

For example, when you have a problem with artifacts, you can look at any example where the name contains 'artifact'

It does not work so well here (which is basically why I did the PR, because I am now used to look at examples to try things ;) )

If there is too much details, maybe we can leave the text but remove the yaml (however, this will be less clear I think)

Anyway, this is your decision, I'll comply :)

Reply:
should you link to the examples?

Reply:
sorry, I do not understand what you mean : you want me to completely remove the link to the jobs example ?

Reply:
I think Yuan is suggestion that you link to more examples?

Reply:
I removed the YAML snippet and only keep the link to the example